src/gsy_e/gsy_e_core/simulation/simulation.py

```diff
@@ -485,7 +485,7 @@ def _execute_simulation(
 
             self._external_events.update(self.area)
 
-            self._compute_memory_info()
+            # self._compute_memory_info()
```

Comment:
Do we have any plans to bring this feature back in the future?

Reply:
Only while debugging memory leaks or other issues. That's why I would recommend to violate our usual rule and leave it commented out here, for convenience.